corehq/apps/enterprise/enterprise.py

```diff
@@ -666,51 +665,31 @@ def rows(self):
 
     @property
     def total(self):
-        total_out_of_date_rows = 0
-        total_last_builds = 0
-
-        for domain in self.account.get_domains():
-            domain_out_of_date_rows, domain_last_builds = self.total_for_domain(domain)
-            total_out_of_date_rows += domain_out_of_date_rows
-            total_last_builds += domain_last_builds
-
-        total_up_to_date = total_last_builds - total_out_of_date_rows
-
-        return _format_percentage_for_enterprise_tile(total_up_to_date, total_last_builds)
+        return '--'
```

Comment:
Can we add a comment explaining why we're skipping this display for now?

Reply:
3a0985a9bbb38580f6bf7dd01267f0902ddd6478